mmap+memset+fork.c

```diff
@@ -17,16 +17,16 @@ int set_size = 4096;
 void usage(){
 	printf("Usage: ./mmap+memset+fork [OPTION]\n");	
 	printf("Tests virtual adress space updates with fork and mmap options\n\n");
-	printf("-d, --debug,		prints out the PID and address touched by memset\n");
-	printf("-k, --keystroke,	same as debug, but requires a char to b entered to show each line\n");
-	printf("-h, --help,		prints out help message\n");
-	printf("-s, --map_shared,	sets MAP_SHARED flag in mmap\n");
-	printf("-p, --map_private,	sets MAP_PRIVATE flag in mmap\n");
-	printf("-f, --fork_child,	causes program to fork a child process\n");
-	printf("-g, --from,		mmaps and memsets from 2^(arg)\n");
-	printf("-t, --till,		mmaps and memsets till 2^(arg)\n");
-	printf("-a, --map_size,		mmaps given size\n");
-	printf("-b, --set_size,		memsets given size\n");
+	printf("-d, --debug		prints out the PID and address touched by memset\n");
+	printf("-k, --keystroke		same as debug, but requires a char to b entered to show each line\n");
```

Comment:
Standard default terminal is only 80 chars wide, my suggestion to be most user friendly is to keep this line inside that length (or make it two lines and push the second line over so that it is justified.)  It would probably be easiest to use spaces instead of tabs since they may be represented differently in the editor vs. the terminal.  (ie, exactly how many spaces is a tab?  in vim you can configure it to any number you want)